### ISSUE TITLE
[SO] Accessibility - detailed view - SO bullets

### ISSUE THREAD

**Issue by PBD-ILR-LU** [informed-governance-project/SERIMA#879]
Release 0.5.18

Within the detailed view of the SO module, the individual security objectives can be accessed by clicking on them.
Moreover, the color of the bullet indicates the filling status of the current SO. The issue is that indicating the filing status just with the color is incompatible with the best practices of the accessibility.
On top of the bullet color, the bullet shape should indicate the filling status:
- not filled - black outline circle and white interior
- partially filled - triangle - orange color
- review failed - square - red color
- fully filled or review OK - round bulled - green (as is)

Contrarily to the color, the shape of the bullet is not self explanatory. The shape and the color should be explained in the legend.

**Comment by jfrocha** [comment]
## Operator view
<img width="1681" height="367" alt="Image" src="https://github.com/user-attachments/assets/8861039d-4dea-499d-87b5-b5c28ae9f049" />

## Regulator view
<img width="1675" height="340" alt="Image" src="https://github.com/user-attachments/assets/8db54cb3-6860-4d16-bbbd-003b8b7811b5" />

**Comment by PBD-ILR-LU** [comment]
For bullets corresponding to "not filled", it would be better to have a thicker outline, like a 2 pixels thick outline.

**Comment by jfrocha** [comment]
<img width="1678" height="283" alt="Image" src="https://github.com/user-attachments/assets/937cfe0e-82d5-43fe-aad9-56290678b338" />
